### ISSUE TITLE
Files with numeric names

### ISSUE THREAD

**Issue by dfcreative** [mattdesl/budo#223]
Eg we have `2298.js`

`budo 2298` gives

```sh
C:\Users\dmitry\AppData\Roaming\npm\node_modules\budo\lib\map-entry.js:16
    parts = file.split(':')
```

whereas `budo 2298.js` is fine.

**Comment by mattdesl** [comment]
Good catch, thanks.

**Comment by mattdesl** [comment]
Fixed in 11.1.2.
